src/components/header/AccountMenuItem.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { SFC } from 'react';
+import styled from '../../styled-components';
+
+import { Link } from '../layout/StyledCopy';
+
+import User from '../icons/User';
+import Book from '../icons/Book';
+import GitBranch from '../icons/GitBranch';
+import Heart from '../icons/Heart';
+import Logout from '../icons/Logout';
+
+interface Props {
+    to?: string;
+    icon: string;
```

Comment:
Why not just simply pass the element here directly, instead of having all this string identifier stuff inbetween?
```
icon: JSX.Element
```

Reply:
Because I wanted the whole icon rendering to be part of the menuItem. But using a separate Icon renderer and pass that as a props would also work. (might be better if we want to reuse these icons somewhere else)